apps/condo/domains/user/components/UserOrganizationsList.tsx

```diff
@@ -106,23 +106,29 @@ const OrganizationEmployeeItem: React.FC<IOrganizationEmployeeItem> = (props) =>
 }
 
 export const UserOrganizationsList = ({ userOrganization, organizationEmployeesQuery }) => {
-    const { objs: userOrganizations, loading } = OrganizationEmployee.useAllObjects(
+    const { objs: userOrganizations, allDataLoaded } = OrganizationEmployee.useAllObjects(
         organizationEmployeesQuery,
         { fetchPolicy: 'network-only' }
     )
 
-    const list = userOrganizations.map((employee, index) => (
-        <OrganizationEmployeeItem
-            employee={employee}
-            key={index}
-            userOrganization={userOrganization}
-        />
-    ))
+    const list = useMemo(() => {
+        return userOrganizations
+            .sort((optionA, optionB) =>
```

Comment:
Can we just move sorting to query / db?

Reply:
No, employees have id in the organization field, but need name